### PR TITLE
Fix JavaScript nits

### PR DESCRIPTION
**Pull request by beaufortfrancois**
@eladalon1983 PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,24 +23,18 @@ It is possible to call `setFocusBehavior()` arbitrarily many times before callin
 ```js
 // Example #1 - unconditional decision.
 
-const controller = new CaptureController;
+const controller = new CaptureController();
```
Comment (eladalon1983) [inline README.md:26]: Thanks for catching.

File: README.md [hunk 2]
```diff
@@ -23,24 +23,18 @@ It is possible to call `setFocusBehavior()` arbitrarily many times before callin
...
 controller.setFocusBehavior("focus-captured-surface");
-const stream = await navigator.mediaDevices.getDisplayMedia({
-  video: true,
-  controller: controller
-});
+const stream = await navigator.mediaDevices.getDisplayMedia({ controller });
```
Comment (eladalon1983) [inline README.md:28]: 1. I want to be explicit; I think it makes things clearer for a wider audience.
2. The `video: true` fix has still not made it into Chrome's active stable revision. People could stumble on this. I'd rather just include it.

File: README.md [hunk 3]
```diff
@@ -107,16 +95,13 @@ controller.setFocusBehavior("focus-captured-surface");
...
+const controller = new CaptureController();
+const stream = await navigator.mediaDevices.getDisplayMedia({ controller });
 
 // This task will be queued behind the the task
 // that closes the window of opportunity to
 // call setFocusBehavior().
 setTimeout(() => {
   controller.setFocusBehavior("focus-captured-surface");
-});
+}, 0);
```
Comment (eladalon1983) [inline README.md:106]: I'm OK with doing this more explicitly, but I wonder - why did you add that, given that your original direction was to be terser? It's an optional parameter defaulting to 0, is it not?

> The time, in milliseconds that the timer should wait before the specified function or code is executed. If this parameter is omitted, a value of 0 is used, meaning execute "immediately", or more accurately, the next event cycle.

Taken from [here](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout).

Comment (beaufortfrancois) [reply README.md:106]: I didn't realize it was optional. 🤦  You're right, let's not add `0`

File: README.md (hunk 2 above)
Comment (beaufortfrancois) [reply README.md:28]: 1. Does not make sense as the wider audience doesn't need to know a `video` attribute.
2. This makes sense to me. Let's not remove `video: true` then.

File: README.md (hunk 1 above)
Comment (beaufortfrancois) [reply README.md:26]: 👍

Comment (beaufortfrancois) [reply README.md:26]: Note that it works though ;)

File: README.md (hunk 2 above)
Comment (eladalon1983) [reply README.md:28]: 1. There are two changes here; the change of `controller: controller` into `controller` and the removal of `video: true`. My first bullet point refers to the former; my second bullet point refers to the latter.
2. Cool.

File: README.md (hunk 3 above)
Comment (eladalon1983) [reply README.md:106]: I'm fine either way. I thought it was common knowledge enough to not require being explicit, but if you were unaware, surely many others don't know that either.

File: README.md (hunk 1 above)
Comment (eladalon1983) [reply README.md:26]: Wait, so...?

Comment (beaufortfrancois) [reply README.md:26]: It is not common to see that. Let's use `new CaptureController();`

Comment (eladalon1983) [reply README.md:26]: Okay.

File: README.md (hunk 2 above)
Comment (beaufortfrancois) [reply README.md:28]: 1. I'd use `controller`. See https://caniuse.com/mdn-javascript_operators_object_initializer_shorthand_property_names

File: README.md (hunk 3 above)
Comment (beaufortfrancois) [reply README.md:106]: Let's not add `0`

Comment (eladalon1983) [reply README.md:106]: Okay.

File: README.md (hunk 2 above)
Comment (eladalon1983) [reply README.md:28]: 1. How sure are we that everyone is aware of this notation?

Comment (beaufortfrancois) [reply README.md:28]: That **everyone** is aware of this notation, zero chance.
However since it has been there since 2015 according to https://caniuse.com/mdn-javascript_operators_object_initializer_shorthand_property_names, I'm confident most web developers know this.

Comment (eladalon1983) [reply README.md:28]: I said "everyone"? Oops. I meant "most readers."
If you think it's commonplace enough, sure, let's do it. But I think `{video: true, controller}` is awkward. But you're the JS guy, I'm the C++ guy, so... I'll take your advice here.

Comment (beaufortfrancois) [reply README.md:28]: If we're keeping `video: true`, then we shouldn't change your code.

**Comment by beaufortfrancois** [comment]
I've opened https://github.com/WICG/conditional-focus/pull/4
